src/app/settings/change-password.component.html

```diff
@@ -2,7 +2,8 @@
   <h1>{{ "changeMasterPassword" | i18n }}</h1>
 </div>
 
-<app-callout type="warning">{{ "loggedOutWarning" | i18n }}</app-callout>
+<bit-callout type="warning">{{ "loggedOutWarning" | i18n }}</bit-callout>
+<!--TODO Update Component Library with enforcedPolicyOptions callout options-->
```

Comment:
Should be a separate component, we don't want to pollute the generic callout with policy knowledge.

Reply:
Creating jira task to add this component! Comment removed.